docs/application/web/api/5.5/device_api/mobile/tizen/mediacontroller.html

```diff
@@ -1691,6 +1747,35 @@ <h4>Attributes</h4>
 </li></ul>
         </div>
 </li>
+<li class="attribute" id="MediaControllerServerInfo::iconURI">
+<span class="attrName"><span class="readonly">                readonly
+</span><span class="type">DOMString </span><span class="name">iconURI</span><span class="optional"> [nullable]</span></span><div class="brief">
+ Server icon URI.
+            </div>
+<p><span class="version">Since: </span>
+ 5.5
+            </p>
+<div class="exceptionlist">
+<p><span class="except">Exceptions:</span></p>
+          <ul class="exception"><li>WebAPIException<ul><li class="list"><p>
+ with error type UnknownError if any error occurs.
```

Comment:
By the way, a minor issue. Please add ,.

with error type UnknownError, if any error occurs.

Reply:
done